src/test/java/uk/gov/hmcts/dts/fact/services/admin/AdminCourtAddressServiceTest.java

```diff
@@ -192,11 +205,41 @@ class AdminCourtAddressServiceTest {
         COUNTY_ID,
         VISIT_OR_CONTACT_US_POSTCODE,
         COURT_SECONDARY_ADDRESS_TYPE_LIST_3,
-        SORT_ORDER_3
+        SORT_ORDER_3,
+        EPIM_ID
     );
+    private static final CourtAddress BAD_EPIM_ADDRESS = new CourtAddress(
+        1,
+        WRITE_TO_US_ADDRESS_TYPE_ID,
+        TEST_ADDRESS1,
+        TEST_ADDRESS_CY1,
+        TEST_TOWN1,
+        null,
+        COUNTY_ID,
+        WRITE_TO_US_POSTCODE,
+        COURT_SECONDARY_ADDRESS_TYPE_LIST,
+        SORT_ORDER_2,
+        BAD_EPIM
+    );
+    private static final CourtAddress NULL_EPIM_ADDRESS = new CourtAddress(
+        1,
+        WRITE_TO_US_ADDRESS_TYPE_ID,
+        TEST_ADDRESS1,
+        TEST_ADDRESS_CY1,
+        TEST_TOWN1,
+        null,
+        COUNTY_ID,
+        WRITE_TO_US_POSTCODE,
+        COURT_SECONDARY_ADDRESS_TYPE_LIST,
+        SORT_ORDER_2,
+        null
```

Comment:
Should we be able to pass in nulls? Interesting. Might be worth pondering

Reply:
nulls are able to pass in so it works if the epim id isn't even passed in at all from admin portal. if the key value pair just doesn't exist in the put request then it'll work anyway and assume mark it as be blank. This is also useful since secondary addresses are still just 'CourtAddress' and they don't have an epim so it handles the fact it doesn't need to exist and it'll still work